### PR TITLE
Pre-commit hooks and installation instructions added.

### PR DESCRIPTION
**Pull request by Stedders**

### CODE REVIEW

**Comment by synek** [comment]
Looks like the CI/CD is having a bit of a moment?

**Comment by Stedders** [comment]
Yeah, some of the initial tests have failed/not-run. There is a second batch that has completed without issue...odd

**Comment by Stedders** [comment]
Should be good now, did a ghost commit and push.

```sh
git commit --allow-empty -m "trigger GitHub actions"
git push
```

File: README.md [hunk 1]
```diff
@@ -55,5 +55,27 @@ A better workflow for git.
...
+
+* Download and try it out
+* Create an [issue](https://github.com/synek/git-plan/issues) if you find a problem or want to discuss something
+* [fork](https://guides.github.com/activities/forking/) the repository, fix a bug or add a feature, and raise a PR
+
+### Development
+
+1. Install the dev requirements `pip install -r requirements_dev.txt`
+2. Install the package in [develop mode](https://pip.pypa.io/en/stable/reference/pip_install/#install-editable) `pip install -e .`
+3. Install the pre-commit hooks `pre-commit install pre-commit` and `pre-commit install pre-push`
```
Comment (synek) [inline README.md:68]: I just tried it out and I needed to add a `-t`, like so:

`pre-commit install -t {pre-commit, pre-push}`

Comment (Stedders) [reply README.md:68]: Let me correct

**Comment by synek** [comment]
Sweet, going to merge this now.

**Pull request merged**